### DISCUSSION TITLE
[Config Support]: Recordings on external USB drive

### DISCUSSION THREAD

**Discussion by Mareckil** [blakeblackshear/frigate#12291] [Config Support]
### Describe the problem you are having

How can I force frigate to save media files on external USB drive?
I am using Home assistant baremetal and Frigate 0.14_beta as addon.
Installed addon Samba NAS and it is mounting my drive as /media/FRIGATE (can't change it to /media/frigate)
When starting frigate it crashes.
So I created symbolic link "frigate" to "FRIGATE" after that Frigate started and was writing on external drive.
But after reboot. It some how created "FRIGATE" folder on internal drive and kept symbolic link "frigate" to it, so frigate started but it was writing on internal drive not on USB.
When I looked on network shares provided by Samba NAS addon I had share "frigate" - my USB drive and share "media" with two folders "FRIGATE" and "frigate" (sybolic link). and those two where on internal drive.
Anybody can help me how to set it up?

### Version

0.14.0-beta3

### Frigate config file

```yaml
empty
```

### Relevant log output

```shell
empty
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

Coral

**Comment by NickM-27** [comment]
There is no officially supported way to do this, with local storage in HA OS. Hopefully home assistant adds support soon

**Reply by Mareckil** [reply]
Well, overdried it moving whole data partition to USB HDD drive, another benefit now HA and Frigate doesn't write so much om internal SSD drive.
